### ISSUE TITLE
Test view reports "There is no data provider registered that can provide view data."

### ISSUE THREAD

**Issue by FrankCoder** [forcedotcom/salesforcedx-vscode#845]
macOS Mojave 10.14.2
VSCode 1.30.1
Salesforce Extension Pack 44.11.0
sfdx-cli/6.44.0-dead729fb5 (darwin-x64) node-v8.9.4

Bringing up the test view in the left panel of VSCode reports: 
"There is no data provider registered that can provide view data."

Clicking the refresh button displays a notification box with the following message:
"command 'sfdx.force.test.view.refresh' not found"

I can list and run tests from the command palette.

### Steps To Reproduce:

I had installed the extensions a while back but this was my first time using it.  I upgraded the CLI to the latest.

I retrieved code from a sandbox org using manifest file project.

I explored things and the test panel was non functional as described above but I could list and run test from the command palette.

### Additional information

![screen shot 2019-01-03 at 10 49 14](https://user-images.githubusercontent.com/627664/50647590-f5f34900-0f46-11e9-93ac-d781238a5404.png)

**Comment by brpowell** [comment]
Hi @FrankCoder. If you have Java 8 installed and you're still getting that error, try adding salesforcedx-vscode-apex.java.home to your settings and specify the runtime path for Java 8. An example is `/Library/Java/JavaVirtualMachines/jdk1.8.0_131.jdk/Contents/Home`

**Comment by FrankCoder** [comment]
Hi @brpowell its the first thing I verified and it was set to this:
   `"salesforcedx-vscode-apex.java.home": "/Library/Java/JavaVirtualMachines/jdk1.8.0_191.jdk",`
I added the /Contents/Home to the end and now everything is working.  

The things is I never manually set that setting so whatever did it, didn't do it right.

I'm closing this bug because it was a side effect of this bad path but I'm wondering if there isn't another problem here.  

Anyway, many thanks!
